packages/format-library/src/text-color/index.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import { RichTextToolbarButton } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import InlineColorUI from './inline';
+
+const name = 'core/text-color';
+const title = __( 'Text Color' );
+
+export const textColor = {
+	name,
+	title,
+	tagName: 'span',
+	className: 'has-inline-color',
+	edit: class TextColorEdit extends Component {
+		constructor() {
+			super( ...arguments );
+
+			this.onClose = this.onClose.bind( this );
+
+			this.state = {
+				isOpen: false,
+			};
```

Comment:
Nit: could be using `useState` to avoid the class component.

Reply:
@youknowriad Just clarifying, I just need to change `isOpen` to `useState` right? Thanks!

Reply:
yes and refactor to be a functional component (hooks don't work on class components)